playback/src/audio_backend/pulseaudio.rs

```diff
@@ -12,39 +12,36 @@ const STREAM_NAME: &str = "Spotify endpoint";
 
 pub struct PulseAudioSink {
     s: Option<Simple>,
-    ss: pulse::sample::Spec,
     device: Option<String>,
     format: AudioFormat,
+    pulse_format: pulse::sample::Format,
 }
 
 impl Open for PulseAudioSink {
     fn open(device: Option<String>, format: AudioFormat) -> Self {
-        info!("Using PulseAudio sink with format: {:?}", format);
+        let mut librespot_format = format;
+        if librespot_format == AudioFormat::F64 {
+            warn!("PulseAudio currently does not support F64 output");
```

Comment:
Note that we will be falling back to F32.

Reply:
Don't really have to `info!("Using PulseAudio sink with format: {:?}", librespot_format);` is right under that?

Reply:
OK

Reply:
Done.